engine/schema/src/main/java/com/cloud/storage/GuestOSVO.java

```diff
@@ -57,6 +57,9 @@ public class GuestOSVO implements GuestOS {
     @Column(name = "is_user_defined")
     private boolean isUserDefined;
 
+    @Column(name = "display", updatable = true, nullable = false)
```

Comment:
can not work on upgrades as it is required during the add guest os calls (before the DB upgrade is done)